CONTRIBUTING.md

```diff
@@ -53,12 +53,12 @@ First of all, thanks for taking the time to contribute! This project can only gr
 
 <hr>
 <h2 id='requestingIcon'>Requesting an Icon</h2>
-<p>To request an icon, you can create an issue in the repository. Please follow these simple guidelines:</p>
+<p>To request an icon, please create an issue in the repository and follow these guidelines:</p>
 <ul>
   <li>Search for other issues already requesting the icon</li>
   <li>If an issue doesn't exist, create an issue naming it "Icon request: <i>name-of-the-icon</i>". </li>
   <li>Please create separate issues for each icon</li>
-  <li><b>Optional:</b> include links where the icon can be found</li>
```

Comment:
I don't think this should be removed. We should know from which sources they took the icon (they could be using a similar or fake icon to the original).

Reply:
That sounds good 👍. I've added the line again.

This is actually listed in the PR template but for some reason, it's not showing for people to use 🤔